.github/workflows/publish-stage.yaml

```diff
@@ -22,14 +22,14 @@ jobs:
         run: npm ci
       - name: Cache Docusaurus
         uses: ./.github/actions/docusaurus-cache
-      - name: Update URL
-        run: 'sed -i ''s!url: "https://docs.camunda.io"!url: "https://stage.docs.camunda.io"!g'' docusaurus.config.js'
```

Comment:
Not needed anymore, because docusaurus.config.js now uses env variables for these values instead.